test/unit/mdc-drawer/modal.foundation.test.js

```diff
@@ -48,7 +48,8 @@ test('#closed untraps the focus when drawer finishes animating close', () => {
 });
 
 test('#handleScrimClick closes the drawer', () => {
-  const {foundation} = setupTest();
+  const foundation = new MDCModalDrawerFoundation();
+  foundation.close = td.func('close');
```

Comment:
...huh. This seems like it shouldn't have been passing without this line...

Reply:
Yeah, I'm a little confused by that too 🤷‍♂️